_posts/ja/newsletters/2021-03-17-newsletter.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Bitcoin Optech Newsletter #140'
+permalink: /ja/newsletters/2021/03/17/
+name: 2021-03-17-newsletter-ja
+slug: 2021-03-17-newsletter-ja
+type: newsletter
+layout: newsletter
+lang: ja
+---
+今週のニュースレターは、失われたLNファンディング・トランザクションの救済に関する議論と、
+リリースとリリース候補の発表および人気のBitcoinインフラストラクチャソフトウェアの注目すべき変更点の通常のセクションを掲載しています。
+
+## ニュース
+
+- **<!--rescuing-lost-ln-funding-transactions-->失われたLNファンディング・トランザクションの救済:**
+  LNのファンディング・トランザクションは、トランザクションのマリアビリティがある場合には安全ではありません。
+  Segwitはほとんどのトランザクションで懸念される第三者によるマリアビリティを排除しましたが、
+  [Replace-by-Fee][topic rbf] (RBF)を使用してファンディング・トランザクションの手数料を引き上げるといった、
+  トランザクションの作成者自信がそのtxidを変異させるケースには対応していません。
+  txidが変異すると、事前に署名された払い戻しトランザクションが無効になるため、ユーザーは自身の資金を取り戻すことができません。
+  さらにリモートノードは自動的にファンディング・トランザクションを確認できないため、
+  資金提供者が資金を取り戻す支援をできない可能性があります。
+
+    今週、Rusty Russellは、この問題を抱えるユーザーが資金を取り戻せるよう、
+    C-Lightningに実装した迅速かつ実験的な機能についてLightning-Devメーリングリストに[投稿しました][russell funding rescue]。
```

Comment:
I'm not sure, but I think `quick' is not the feature itself, but for its implementation period ( just a day from issue opened).

```suggestion
    C-Lightningに素早く実装した実験的な機能についてLightning-Devメーリングリストに[投稿しました][russell funding rescue]。
```